CHANGES.rst

```diff
@@ -1,8 +1,20 @@
 Changelog
 =========
 
-3.0.1 (unreleased)
-------------------
+3.1 (unreleased)
+----------------
+
+- Plone 5 + Plone 4 compatibility.
+  [thet]
+
+- Extend the ``uninstall`` profile with more de-registrations.
+  [thet]
+
+- Add a ``available`` method to the transform, which can be overridden to e.g. allow the transform also for non-anonymous users.
+  [thet]
+
+- Fix the blacklist xpath selector to also match elements with the class directly set on it.
```

Comment:
I think it would be sane to split this into a different PR so it will be easier for you to do the rebase now; we can also get rid of the functional tests there.